app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -2,23 +2,47 @@
 
 module Jobs
   class EmitWebHookEvent < Jobs::Base
+    # Required option:
+    #   model                   - The symbol represents the model class
+    #   load_record             - A lambada for loading the records
+    #
+    # Acceptable options:
+    #   serializer_name         - A serializer class name otherwise `<Model>Serializer` is used.
+    def self.web_hook_serialization_properties
+      @@web_hook_serialization_properties ||= [
+        { model: :topic, load_record: ->(topic_id) { TopicView.new(topic_id, Discourse.system_user) }, serializer_name: "WebHookTopicViewSerializer" },
```

Comment:
I'll prefer if we define these a individual methods. Much easier to test if we want to.